examples/ex19.cpp

```diff
@@ -0,0 +1,239 @@
+#include "mfem.hpp"
```

Comment:
- Add LLNL copyright banner and style it as the rest of the MFEM examples.
- Change the number to ex18.